src/main/java/dev/goral/rpgmanager/user/User.java

```diff
@@ -125,3 +125,4 @@ public final int hashCode() {
 
 
 }
+//TODO: Dodać obsługę zdjęcia profilowego
```

Comment:
to dosłownie jedna linijka :< w sensie żeby dodać ją do logiki (pole + odpowiednio konstruktory)